src/main/webapp/view/view.js

```diff
@@ -45,6 +46,7 @@ async function initialize() {
   setLectureName();
   window.video.loadVideoApi();
   intializeDiscussion();
+  IconFeedback.loadIconFeedbackList();
```

Comment:
is this just here for testing?  won't all the feedback stuff be on a different page?

Reply:
Sorry, I wasn't finished with this PR at the time of review